arm/Microsoft.MachineLearningServices/workspaces/deploy.bicep

```diff
@@ -153,6 +160,25 @@ resource workspace 'Microsoft.MachineLearningServices/workspaces@2021-04-01' = {
   }
 }
 
+module workspace_computes 'computes/deploy.bicep' = [for compute in computes: {
+  name: '${workspace.name}-${compute.name}-compute'
+  params: {
+    machineLearningWorkspaceName: workspace.name
+    name: compute.name
+    location: location
+    sku: sku
+    identity: identity
+    tags: tags
+    deployCompute: compute.deploy
+    computeLocation: compute.location
+    computeDescription: compute.description
```

Comment:
If you specify it like this, `description` will become a required parameter even though the child module does not declare it as such.

The recommendation is (and the same accounts for all other properties): If a parameter is optional, use the syntax: `description: contains(compute, 'description') ? compute.description :  ''`
In words: If the compute object contains the property, use it, otherwise use the default specified in the child itself.

Unfortunately, there is no feature in bicep that it would automatically pick up the child default (like a special keyword)

Reply:
@MrMCake Updated.